src/main/java/bookstore/service/BookServiceImpl.java

```diff
@@ -44,11 +45,19 @@ public void deleteById(Long id) {
 
     @Override
     public BookDto update(CreateBookRequestDto bookRequestDto, Long id) {
-        Book book = (Book) bookRepository.getBookById(id).orElseThrow(
+        Book book = bookRepository.getBookById(id).orElseThrow(
                 () -> new EntityNotFoundException("Can`t find book by id " + id)
         );
         bookMapper.updateBook(bookRequestDto, book);
         book.setId(id);
```

Comment:
this line should be deleted

Reply:
not fixed! Please fix it before merging;)